docs/source/contributing/dev-contributions/contrib_guide.rst

```diff
@@ -121,6 +121,7 @@ including partial or in-progress work, is appreciated.
 Contribution Workflow
 ---------------------
 .. image:: ../../_static/_images/contribution_workflow.png
+   :alt: 
```

Comment:
```suggestion
   :alt: A flow chart with 13 labeled boxes linked by arrows. The chart is uni-directional.  At each step, arrows point forward to one or more boxes and back to the previous box or boxes. Here the flow chart is described as lists in which the possible next steps are listed beneath each box label.

1. Start
- forward to 'Find an issue to address'
2. Find an issue to address
- forward to 'Identify the relevant section of the codebase'
3. Identify the relevant section of the codebase
- forward to 'Write simple test cases to verify that your fix or enhancement works.'
4. Write simple test cases to verify that your fix or enhancement works.
- forward to 'Implement the bug fix or enhancement in the appropriate location'
5. Implement the bug fix or enhancement in the appropriate location.
- Forward to 'Run the test suite and verify that the updated code works appropriately.'
6. Run the test suite and verify that the updated code works appropriately.
- lateral to 'Test suit passes!'
- lateral to 'Test suit fails!'
7. Test suit fails
```

Reply:
Here's an alternative alt-text. A short one, and refer to link above the image for the full, long alt-text.

Reply:
This one may be too long. I have commented below an alternative.